pci-dss/deny-ingress-creation.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-ingress-creation
+  annotations:
+    policies.kyverno.io/title: deny Ingress Creation
+    policies.kyverno.io/category: Networking
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Ingress
+    policies.kyverno.io/description: >-
+      Ingress provides routing rules to manage access to the services within a Kubernetes cluster. 
+      This policy blocks the creation of Kubernetes ingress resources.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+  - name: deny-ingress-creation
+    match:
```

Comment:
latest Kyverno expects syntax in the form of 
```
match:
  any:
  - resources:
```

Reply:
@vikashkaushik01 can you test this in the latest kyverno version, please?